src/bootstrap/lib.rs

```diff
@@ -1431,6 +1431,13 @@ impl Build {
             return Vec::new();
         }
 
+        if !stamp.exists() {
+            eprintln!(
+                "Warning: Unable to find the stamp file, did you try to keep a nonexistent build stage?"
+            );
+            crate::detail_exit(1);
```

Comment:
I have two comments here:
1. This should say "error", not "warning"
2. This should print the file path of the stamp file; currently it's a regression compared to the existing error message.

@Teapot4195 do you think you could make a follow-up PR with both those changes?

Reply:
Yeah, I'll make a follow up PR for those changes

Reply:
Thank you!